### PR TITLE
Add Thing Model text

### PR DESCRIPTION
**Pull request by sebastiankb**
see subject

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-architecture/pull/540.html" title="Last updated on Oct 5, 2020, 2:05 PM UTC (37a6c6e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-architecture/540/466b34c...37a6c6e.html" title="Last updated on Oct 5, 2020, 2:05 PM UTC (37a6c6e)">Diff</a>

### CODE REVIEW

**Comment by sebastiankb** [comment]
the content shall be aligned with https://github.com/w3c/wot-thing-description/pull/938

**Review by relu91 — APPROVED** [review]

**Comment by sebastiankb** [comment]
also see https://w3c.github.io/wot-thing-description/#thing-model

File: index.html [hunk 1]
```diff
@@ -2662,9 +2662,22 @@ <h2>WoT Thing Description</h2>
 
         <section id="thing-model">
             <h2>Thing Model</h2>
-            <section class="ednote">
-                TODO: Create text that introduces the concepts of a thing model.
-            </section>
+   
+		<p>The <a>Thing Model</a> offers the oppertunity to define a template-based model for
```
Comment (danielpeintner) [inline index.html:2666]: Typo: opportunity

File: index.html [hunk 2]
```diff
@@ -2662,9 +2662,22 @@ <h2>WoT Thing Description</h2>
 
         <section id="thing-model">
             <h2>Thing Model</h2>
-            <section class="ednote">
-                TODO: Create text that introduces the concepts of a thing model.
-            </section>
+   
+		<p>The <a>Thing Model</a> offers the oppertunity to define a template-based model for 
+		    <a>Thing Description</a> instances. The peculiarity is that a <a>Thing Model</a> has no
```
Comment (danielpeintner) [inline index.html:2667]: I am not native speaker but I haven't seen "peculiarity" being used a lot.

maybe: (In contrast to a TD,) the characteristic of Thing Model is that it has no ..

File: index.html [hunk 3]
```diff
@@ -2662,9 +2662,22 @@ <h2>WoT Thing Description</h2>
...
+		    <a>Thing Description</a> instances. The peculiarity is that a <a>Thing Model</a> has no 
+			specific or only partial communication- and  security-based information. This kind 
+			of missing information is supplemented by the creation of <a>Thing Description</a> 
+			instantiation. 
+		</p>
+		
+		<p>
+	        A <a>Thing Model</a> mainly describes the <a>Properties</a>, <a>Actions</a>, and <a>Events</a> and 
+                common metadata which then should be valid for all instantiated <a>Thing Descriptions</a>. This paradigm 
+                can be compared with class definition (~Thing Model) in object-oriented programming to create objects (~Thing Descriptions).
```
Comment (danielpeintner) [inline index.html:2676]: "class definitions" -> "class definitions" ?

File: index.html [hunk 4]
```diff
@@ -2662,9 +2662,22 @@ <h2>WoT Thing Description</h2>
...
+		Such <a>Thing Models</a> gets relevant for, e.g., mass production of IoT devices, onboard scenarios such in cloud
```
Comment (danielpeintner) [inline index.html:2677]: "gets" --> "get"

**Comment by mlagally** [comment]
approved in arch call on 5.10.

**Pull request merged**